protocol-units/da/movement/protocol/util/src/blob/ir/data.rs

```diff
@@ -80,4 +80,28 @@ pub mod block {
 			Ok(Self::now(blob))
 		}
 	}
+
+	impl<C> TryFrom<block::Id> for InnerSignedBlobV1Data<C>
+	where
+		C: Curve + Verify<C> + Digester<C>,
+	{
+		type Error = anyhow::Error;
+
+		fn try_from(id: block::Id) -> Result<Self, Self::Error> {
+			let blob = id.as_bytes().to_vec();
+			Ok(Self::now(blob))
+		}
+	}
+
+	impl<C> TryFrom<Vec<block::Id>> for InnerSignedBlobV1Data<C>
+	where
+		C: Curve + Verify<C> + Digester<C>,
+	{
+		type Error = anyhow::Error;
+
+		fn try_from(ids: Vec<block::Id>) -> Result<Self, Self::Error> {
+			let blob = bcs::to_bytes(&ids)?;
+			Ok(Self::now(blob))
+		}
+	}
```

Comment:
These conversions look suspicious next to `TryFrom<Block>`, and they were previously removed. Is it really so that a blob can encode a block, a single ID, or an array of IDs, with nothing to discriminate?
The change also does not seem to belong with the rest of the PR. A stray checkout, perhaps?

Reply:
I remove it in another branch that has been merged in main.